lib/process-services/process-list/components/start-process.component.ts

```diff
@@ -96,37 +96,41 @@ export class StartProcessInstanceComponent implements OnChanges {
             this.appId = changes['appId'].currentValue;
         }
 
+        if (changes['processDefinitionId'] && changes['processDefinitionId'].currentValue) {
```

Comment:
Why do you need this? processDefinitionId being an input parameter itself, I suppose it should be updated the same way in the background by angular. Am I wrong?

Reply:
In addition, the input is not processDefinitionId but **processDefinition**